.travis.yml

```diff
@@ -25,11 +25,13 @@ env:
     - DEP_RELEASE_TAG="v0.4.1"
 
 install:
-  - npm install -g npm@5.5.1 yarn@1.3.2
   - sudo apt install -y cvs
   - eval "$(gimme)"
   - curl https://raw.githubusercontent.com/golang/dep/7d5cd199ce454707f81c63b7ea4299151b8b981d/install.sh | sh
+  - npm install -g npm@5.5.1 yarn@1.3.2
```

Comment:
Any reason to move this line down?

Reply:
Only to "sort of" sort it alphabetically WRT to `phpenv`.